build/gulpfile.vscode.js

```diff
@@ -122,7 +122,7 @@ const optimizeVSCodeTask = task.define('optimize-vscode', task.series(
 			},
 			manual: [
 				{ src: [...windowBootstrapFiles, 'out-build/vs/code/electron-sandbox/workbench/workbench.js'], out: 'vs/code/electron-sandbox/workbench/workbench.js' },
-				{ src: [...windowBootstrapFiles, 'out-build/vs/code/electron-sandbox/issue/issueReporter.js'], out: 'vs/code/electron-sandbox/issue/issueReporter.js' },
+				{ src: [...windowBootstrapFiles, 'out-build/vs/workbench/contrib/issue/electron-sandbox/issueReporter.js'], out: 'vs/vs/workbench/contrib/issue/electron-sandbox/issueReporter.js' },
```

Comment:
Please create a new issue and reference it here in a TODO so that we do not forget to remove this line when the issue reporter uses `window.open` on desktop too.